app/index.tsx

```diff
@@ -156,4 +157,4 @@ class App extends React.Component<{}, State> {
   }
 }
 
-ReactDOM.render(<App />, document.getElementById("app"));
+ReactDOM.render(<><GlobalStyle/><App /></>, document.getElementById("app"));
```

Comment:
To jest super. Ja nigdy tego nie robiłam. Nie wiedziałam, że się tak robi z stylami do body ;)